pkg/stanza/tokenize/multiline_test.go

```diff
@@ -176,11 +287,27 @@ func TestLineEndSplitFunc(t *testing.T) {
 				"log2 LOGEND",
 			},
 		},
+		{
+			Name:                  "TwoLogsLineEndSimpleOmitPattern",
+			Pattern:               `LOGEND$`,
+			OmitPatternFromRecord: true,
+			Input:                 []byte("log1 LOGEND LOGEND\nlog2 LOGEND\n"),
+			ExpectedTokens: []string{
+				"log1 LOGEND",
+				"log2",
+			},
+		},
```

Comment:
Similarly, the same but without the `$` would be interesting because of the empty token:
```
ExpectedTokens: []string{
	"log1",
	"",
	"log2",
},
```